examples/ads.amp.html

```diff
@@ -265,6 +266,12 @@
       data-host="talos.adman.gr">
   </amp-ad>
 
+  <h2>AdmanMedia</h2>
+  <amp-ad width="300" height="250"
```

Comment:
I get error when testing locally. Could you please take a look? 
![image](https://cloud.githubusercontent.com/assets/5419904/26224327/ae705be4-3bd6-11e7-8839-d52aeccee559.png)